lib/python/rose/meta_type.py

```diff
@@ -206,15 +206,15 @@ def transform(self, value):
 
 
 def meta_type_checker(value, meta_type):
-    c = MetaType.get_meta_type(meta_type)
-    c = c()
-    return c.is_valid(value)
+    item = MetaType.get_meta_type(meta_type)
+    item = item()
+    return item.is_valid(value)
 
 
 def meta_type_transform(value, meta_type):
-    c = MetaType.get_meta_type(meta_type)
-    c = c()
+    item = MetaType.get_meta_type(meta_type)
+    item = item()
     try:
-        return c.transform(value)
+        return item.transform(value)
     except Exception:
```

Comment:
Is a more specific exception class possible here - ``except NotImplementedError:`` maybe?

Reply:
I think `item.transform` may do anything.